src/set_resources.py

```diff
@@ -31,12 +32,12 @@ def generate_empty_file(name):
 
 def error_message(index, e_type):
     if e_type == 'ip':
-        print "Error on line {}. IP is not valid.".format(index)
+        logging.warning("Error on line {}. IP is not valid.".format(index))
     elif e_type == 'value':
-        print "Error on line {}. Resource values must be integers.".format(index)
+        logging.warning("Error on line {}. Resource values must be integers.".format(index))
     elif e_type == 'cpu':
-        print "Error on line {}. CPU values must be between 0 and 100. (100 as MAX CPU)".format(index)
-    print "Exiting script"
+        logging.warning("Error on line {}. CPU values must be between 0 and 100. (100 as MAX CPU)".format(index))
+    logging.warning("Exiting script")
```

Comment:
Should be at error level

Also, why doesn't this exit after saying it's exiting?